src/renderer/components/settings-general-font.tsx

```diff
@@ -51,12 +51,13 @@ export class FontSettings extends React.Component<
   }
 
   /**
-   * Handles a change in the editor font family.
+   * Handles a change in the editor font size.
    *
    * @param {React.FormEvent<HTMLInputElement>} event
    */
   public handleSetFontSize(event: React.FormEvent<HTMLInputElement>): void {
-    const fontSize = parseInt(event.currentTarget.value, 10);
+    const parsedFontSize = parseInt(event.currentTarget.value, 10);
+    const fontSize = isNaN(parsedFontSize) ? undefined : parsedFontSize;
```

Comment:
```suggestion
    const fontSize = Number.isNaN(parsedFontSize) ? undefined : parsedFontSize;
```

```js
isNaN(undefined) // => true
Number.isNaN(undefined) // => false
```

Although parseInt does not return undefined, I recommend avoiding isNaN at all times for better code robustness.